src/main/java/com/jcabi/log/VerboseProcess.java

```diff
@@ -433,6 +485,9 @@ public Void call() throws Exception {
                     new OutputStreamWriter(this.output, VerboseProcess.UTF_8)
                 );
                 try {
+                    StringBuilder sb = new StringBuilder();
+                    String previousLine = null;
```

Comment:
@dean-e-clark again, this PR cannot have `null` at all, it's  billion-dollar mistake :wink: